test_ViewModel.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from datetime import datetime, timedelta
+
+
+from model import Item, ViewModel
+
+def test_items():
+    assert len(get_test_items()) == len(get_test_view_model().items)
+    
+
+def test_to_do_items():
+    assert len(get_test_view_model().todoItems) == 1
+
+def test_to_do_items_values():
+    for item in get_test_view_model().todoItems:
+        id, status, title = item.id, item.status, item.title
+        assert id == '5f5a4c01846bb381f6f3fa8b' and status == 'To Do' and title == 'To Do 111'
+
+def test_doing_items():
+    assert len(get_test_view_model().doingItems) == 1
+
+def test_doing_items_values():
+    for item in get_test_view_model().doingItems:
+        id, status, title = item.id, item.status, item.title
+        assert id == '5f6f50a13a48c95efb5679f1' and status == 'Doing' and title == 'To Do 222'
+        
+def test_done_items():
+    assert len(get_test_view_model().doneItems) == 2
+
+def test_done_items_values():
+    for item in get_test_view_model().doneItems:
+        id, status, title = item.id, item.status, item.title
+        assert id == '5f6f50cc2316514d6241e6ba' and status == 'Done'
+        #assert id == '5f6f50cc2316514d6241e6ba' and status == 'Done' and title == 'To Do 333'
```

Comment:
You can combine these two tests.

The second test would pass if .doneItems was an empty list.

Also it's a bit weird to have the same id for two different items in the list. Why not just assert values for the first item and then assert values including a different id/name for the second item?